cmd/d2/main.go

```diff
@@ -37,6 +37,7 @@ func run(ctx context.Context, ms *xmain.State) (err error) {
 	bundleFlag := ms.FlagSet.BoolP("bundle", "b", true, "bundle all assets and layers into the output svg")
 	versionFlag := ms.FlagSet.BoolP("version", "v", false, "get the version and check for updates")
 	debugFlag := ms.FlagSet.BoolP("debug", "d", false, "print debug logs")
+	sketchFlag := ms.FlagSet.BoolP("sketch", "s", false, "use the sketch plugin, making diagrams look sketched by hand")
```

Comment:
fyi @nhooyr, revisiting this, now that sketch is not a plugin, it should be treated as a boolean flag because it's a replacement for the default render. Back when it was a plugin, we drew the default, and then targeted with regex to replace. Now that it's not a plugin, we can just draw it as desired the first time.

The pipeline mechanism still makes sense to me for post-render steps, like the "high-contrast" or "colorblind" as discussed. But for sketch, I'm just going to make it a boolean flag.

Reply:
Sketch can still be made a plugin to avoid having to pull in an entire JS engine for builds only using TALA.

I think it's ok to make it a flag for now, makes it much easier to use.